ui/src/app/components/inputs.tsx

```diff
@@ -457,9 +457,17 @@ export class CheckBox extends React.Component<CheckBoxProps, CheckBoxState> {
 export const Select = ({ value, options, onChange, ...props }) => {
   return (
     <RSelect
+      classNamePrefix='react-select'
       value={options.find((o) => o.value === value)}
       options={options}
       onChange={(o) => onChange(o?.value)}
+      styles={{
+        input: (baseStyles) => ({
```

Comment:
ReactSelect needs to be updated in three places in order to get the input to size how we would like. This currently only sizes the outer container. That is labelled "A" in this diagram.

![Screen Shot 2023-02-10 at 1 42 57 PM](https://user-images.githubusercontent.com/24514630/218174328-ce95a638-1507-457e-bc4f-86d28fe65f8d.png)